packages/print-styles/src/components/signature-block.css

```
@@ -0,0 +1,22 @@
+@media print {
+  .pdf-signature-block .markdown-preview-sizer::after {
+    content: "";
```

Stylelint (17.3.0)

[error] Unexpected duplicate "content" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)